src/Kunstmaan/MediaBundle/Resources/views/Folder/show.html.twig

```diff
@@ -217,6 +216,22 @@
                             </div>
                         </div>
                     </div>
+                    <div class="row list-group">
```

Comment:
Maybe put this in a general template that is included where needed. That removes duplicate code.